_posts/en/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,270 @@
+        money to Alice.
+
+    - *Bitcoin Core not in charge of network:* Gloria Zhao [wrote][zhao
+      no control] in the IRC discussion, "I think whatever option we
+      take, it should be made abundantly clear to users that Core does
+      not control whether full RBF happens or not. We could revert
+      [25353][bitcoin core #25353] and it could still happen. [...]"
+
+    - *No removal means problem could happen:* in the IRC discussion,
+      Anthony Towns [echoed][towns uncoordinated] his points from last
+      week, "if we're not going to remove the mempoolfullrbf option from
+      24.0, we're going for an uncoordinated deployment."
+
+        Greg Sanders was [doubtful][sanders doubt], "the question is
+        will 5%+ set a variable. I suspect not."  Towns [replied][towns
+        uasf], "[UASF][topic soft fork activation] `uacomment`
+        demonstrated it's easy to get ~11% to set a variable in just a
+        couple of weeks".
+
+    - *Should be an option:* Martin Zumsande [said][zumsande option] in
+      the IRC discussion, "I think that if a meaningful number of node
+      operators and miners want a specific policy, it shouldn't be on
+      the devs to tell them 'you can't have that now'. Devs can and
+      should give a recommendation (by picking the default), but
+      providing options to informed users should never be a problem."
+
+    As of this writing, no clear resolution had been reached.  The
+    `mempoolfullrbf` option is still included in the release candidates
+    for the upcoming version of Bitcoin Core 24.0 and it is Optech's
+    recommendation that any service depending on zero conf transactions
+    carefully evaluate the risks, perhaps starting by reading the emails
+    linked in [last week's newsletter][news222 rbf].
+
+- **CoreDev.tech transcripts:** prior to The Atlanta Bitcoin Conference
+  (TabConf), several developers participated in a CoreDev.tech event.
+  [Transcripts][coredev xs] of several of the meetings from the event
+  have been provided by Bryan Bishop.  Notable discussions included:
+
+    - [Transport encryption][p2p encryption]: a conversation about the
+      recent update to the [version 2 encrypted transport
+      protocol][topic v2 p2p transport] proposal (see
+      [Newsletter #222][news222 bip324]).  This protocol would make it
+      harder for network eavesdroppers to learn which IP address
+      originated a transaction and improve the ability to detect and
+      resist man-in-the-middle attacks between honest nodes.
+
+        The discussion covers several of the protocol design
+        considerations and is a recommended read for anyone wondering
+        why the protocol authors made certain decisions.  It also
+        examines the relationship to the earlier [countersign][topic
+        countersign] authentication protocol.
+
+    - [Fee market][]: a wide-ranging discussion about transaction fees
```

Comment:
The good being bid on is blockspace, not fees. So isn't it a “blockspace market” rather than a fee market?

```suggestion
    - [Blockspace market][]: a wide-ranging discussion about transaction fees
```